usecases/linearFunction/linear_model.py

```diff
@@ -34,6 +35,18 @@ def __call__(self, parameters):
         df_x = parameters['b'] * np.ones(len(self.x_derivative))
         return [f_x, df_x]
 
+    def get_parameter_dict(self):
+        """Create a parameter list initialized with parameters all set either to None
+        or to some standard value (in case this is a deterministic variable, this value will
+        always be used
+
+        Returns: parameter list
+        """
+        prm = ModelParameters()
+        prm.define("a", None)
+        prm.define("b", None)
```

Comment:
I first thought about just returning a list of ["a", "b"], but the actual ModelParameters with None is very nice. Would you agree if I, over at BayesianInference, change default value of
`def define(self, name, value=0)`
to
`def define(self, name, value=None)`
?

Reply:
Yes, that prevents using variables that are not initialized.